src/Analyzers/CSharp/Tests/PopulateSwitch/PopulateSwitchStatementTests_FixAllTests.cs

```diff
@@ -271,7 +269,7 @@ void Method()
                     }
                 }
             }
-                    </Document>
+            </Document>
```

Comment:
What caused this indentation change?

Reply:
The doc is getting formatted, and tha tcleans up the errant final whitespace we weren't touching before.